pandas/tests/tools/test_concat.py

```diff
@@ -1662,6 +1662,42 @@ def test_union_categoricals_ordered(self):
         with tm.assertRaisesRegexp(TypeError, msg):
             union_categoricals([c1, c2])
 
+    def test_union_categoricals_ignore_order(self):
+        c1 = Categorical([1, 2, 3], ordered=True)
```

Comment:
add the issue number as a comment. pls add a tests with ``ignore_order=False`` (explicity set), and additional one with no ``ignore_order`` kw passed (to tests the defaults)